src/confd/src/ietf-syslog.c

```diff
@@ -109,8 +115,10 @@ static size_t selector(sr_session_ctx_t *session, struct action *act)
 
 		fprintf(act->fp, "%s%s.%s", i ? ";" : "", fxlate(facility), sxlate(severity));
 
-		free(facility);
-		free(severity);
+		if (facility)
```

Comment:
Put this check after this line instead:
facility = srx_get_str(session, "%s/facility", entry->xpath);
 why continue to write config after that if null?

Reply:
OK